CorsixTH/Lua/world.lua

```diff
@@ -647,6 +647,7 @@ function World:tickEarthquake()
 
       self.next_earthquake.remaining_damage = self.next_earthquake.remaining_damage - 1
       self.next_earthquake.damage_timer = self.next_earthquake.damage_timer + earthquake_damage_time
+      self.next_earthquake.machwarn_trigger = false
```

Comment:
You added a new variable, its meaning should be documented (see near line 101 in `world.lua`).